Dockerfile

```diff
@@ -13,96 +8,64 @@ COPY modules/ modules/
 RUN find modules -type f ! \( -name Gemfile -o -name "*.gemspec" -o -path "*/lib/*/version.rb" \) -delete && \
     find modules -type d -empty -delete
 
-###
-# shared build/settings for all child images, reuse these layers yo
-###
-FROM rubyimg AS base
+FROM rubyimg
+
+# Allow for setting ENV vars via --build-arg
+ARG BUNDLE_ENTERPRISE__CONTRIBSYS__COM \
+  RAILS_ENV=development \
+  USER_ID=1000
+ENV RAILS_ENV=$RAILS_ENV \
+  BUNDLE_ENTERPRISE__CONTRIBSYS__COM=$BUNDLE_ENTERPRISE__CONTRIBSYS__COM \
+  BUNDLER_VERSION=2.4.9
+
+RUN groupadd --gid $USER_ID nonroot \
+  && useradd --uid $USER_ID --gid nonroot --shell /bin/bash --create-home nonroot --home-dir /app
+
+WORKDIR /app
```

Comment:
Does this change need an accompanying change in the devops repo like we did for GIBCT?

Reply:
No, I'm still using `VETS_API_USER_ID` variable from the DevOps repo and set the variable in the review instance docker-compose file